ts/protocol/utils.ts

```diff
@@ -132,7 +132,9 @@ export function stringValidator(value: any): boolean {
  * Returns true if the current origin is https.
  */
 export function isSecureOrigin(): boolean {
-  return window.location.protocol === 'https:';
+  return window.location.protocol === 'https:' ||
```

Comment:
It is unrelated to the Request Context indeed, but I mention it in the PR description. The PR was pretty small so I wanted to make that change here, is that OK?